roles/openshift_node/tasks/storage_plugins/iscsi.yml

```diff
@@ -22,12 +22,16 @@
     - rpcbind
     - iscsid
 
+# Copy conf file only if not exists
 - name: Template multipath configuration
+  stat:
+    path: "/etc/multipath.conf"
+  register: st
   template:
```

Comment:
Its easier to set `force: no` here - it would only template this if the file doesn't exist, see https://docs.ansible.com/ansible/latest/modules/template_module.html

Reply:
@vrutkovs  nice - updating